src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CreateCollectionOperation.java

```diff
@@ -149,6 +158,21 @@ private Uni<Supplier<CommandResult>> executeCollectionCreation(QueryExecutor que
     return indexResult.onItem().transform(res -> new SchemaChangeResult(res));
   }
 
+  void verifyLimits(List<Schema.CqlTable> tables) {
+    final int MAX_COLLECTIONS = dbLimitsConfig.maxCollections();
+    if (tables.size() >= MAX_COLLECTIONS) {
+      throw new JsonApiException(
+          ErrorCode.TOO_MANY_COLLECTIONS,
+          String.format(
+              "%s:  number of Collections cannot exceed %d, already have %d",
```

Comment:
Why uppercase "C" in "Collections"?

Also, would be nice to specify "number of collections in database cannot exceed 5", because this collection limit is database-specific, not global.

Reply:
Thank, I will see how other error messages do capitalization of "collections" and follow that (i.e. capitalize if already doing that; otherwise not). And will add reference to database.

Reply:
Ok yes, no capitalization used for "collection", removed: added ref to "in databse".